src/vorta/store/migrations.py

```diff
@@ -15,7 +17,7 @@
 )
 
 
-def run_migrations(current_schema, db_connection):
+def run_migrations(current_schema: Any, db_connection: SqliteDatabase):
```

Comment:
Do we have to specify `Any`? Isn't it the default, implicit type hint?

Reply:
I think I left that one undone. It should be `SchemVersion` from `vorta.store.models`